ConnectionRestApiClients/clients/csharp/examples/ImportIOM/ImportIOM.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
net6.0 will be depreciated soon. depreciated soon.
We should use net8.0 (or  <TargetFrameworks>net6.0;net48</TargetFrameworks> to have examples for the old framework)

Reply:
Changed to .Net8.0